lib/Template/nodes/TemplateNodeComponent.js

```diff
@@ -17,6 +18,39 @@ export class TemplateNodeComponent extends TemplateNode {
      * @returns {string}
      */
     compile() {
+        return this._compile(this.getPragma());
+    }
+
+    /**
+     * @param {{}} pragma
+     * @returns {string}
```

Comment:
it can return promise or string, because of str:42 (_compileAsync returns promise|string too)